VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionPostsAdapter.java

```diff
@@ -39,11 +45,21 @@ public void render(BaseViewHolder tag, DiscussionThread discussionThread) {
 
         {
             Icon icon = FontAwesomeIcons.fa_comments;
+            @ColorInt
+            final int iconColor;
```

Comment:
The default `iconColor` value should be initialized along with the default `icon` value; either they should both be initialized to their default values at the start, or they should both be initialized in the `else` block.